specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -1966,6 +2593,30 @@
       "type": "array",
       "uniqueItems": true
     },
+    "TranscriptionPiiCategories": {
+      "description": "(Optional) Describes the PII categories to return for detection. By default default categories will be returned which will vary with languages.",
```

Comment:
"Optional" on the model doesn't make much sense, IMO. Any properties of this type may be optional, but not the type itself. I would just elide it.

Reply:
sure.